tests/h/conftest.py

```diff
@@ -246,6 +246,7 @@ def pyramid_request(db_session, fake_feature, pyramid_settings):
     request.params = MultiDict()
     request.GET = request.params
     request.POST = request.params
+    request.user = None
```

Comment:
I think it would probably be a good idea to have some separation between the request methods/properties which are part of Pyramid but are being overridden vs those which are custom in our code here.